static/css/nl_interface.scss

```diff
@@ -193,9 +193,13 @@ $answer-min-height: 100vh;
       padding: 0.25rem;
     }
 
+    .map-with-margin {
+      margin-right: 15px;
+      height: 100%;
```

Comment:
do we need this `height:100%?` isn't height always going to be 100% with or without the ranking section

Reply:
When height isn't set here, the maps on the place pages don't take the full height (instead, they're stuck at 160px min height).

Reply:
Played around with this more: now that I removed the extra div, height is no longer set here.